src/ai/backend/manager/api/filebrowser.py

```diff
@@ -0,0 +1,149 @@
+        )
+    proxy_name, _ = root_ctx.storage_manager.split_host(host)
+    try:
+        proxy_info = root_ctx.storage_manager._proxies[proxy_name]
+    except KeyError:
+        raise InvalidArgument("There is no such storage proxy", proxy_name)
+    headers = {}
+    headers["X-BackendAI-Storage-Auth-Token"] = proxy_info.secret
+    try:
+        async with proxy_info.session.request(
+            "POST",
+            proxy_info.manager_api_url / "storage/filebrowser/create",
+            headers=headers,
+            json={"host": host, "vfolders": vfolders},
+        ) as client_resp:
+            print(headers, host, proxy_info.manager_api_url / "storage/filebrowser/create")
+            print(web.json_response(await client_resp.json()))
+            return web.json_response(await client_resp.json())
+    except aiohttp.ClientResponseError:
+        raise
+
+
+@auth_required
+@server_status_required(READ_ALLOWED)
+@check_api_params(
+    t.Dict(
+        {
+            t.Key("container_id"): t.String,
+        },
+    ),
+)
+async def destroy_filebrowser(
+    request: web.Request,
+    params: Any,
+) -> web.Response:
+    root_ctx: RootContext = request.app["_root.context"]
+    container_id = params["container_id"]
+
+    volumes = await root_ctx.storage_manager.get_all_volumes()
+
+    # search for volume among available volumes which has file browser container id in order to destroy
+    for volume in volumes:
+        proxy_name = volume[0]
+        try:
+            proxy_info = root_ctx.storage_manager._proxies[proxy_name]
+        except KeyError:
+            raise InvalidArgument("There is no such storage proxy", proxy_name)
+
+        headers = {}
+        headers["X-BackendAI-Storage-Auth-Token"] = proxy_info.secret
+        auth_token = proxy_info.secret
+        try:
+            async with proxy_info.session.request(
+                "DELETE",
+                proxy_info.manager_api_url / "storage/filebrowser/destroy",
+                headers=headers,
+                json={"container_id": container_id, "auth_token": auth_token},
+            ) as client_resp:
+                return web.json_response(await client_resp.json())
+        except aiohttp.ClientResponseError:
+            raise
+    return web.json_response({"status": "fail"})
+
+
+async def init(app: web.Application) -> None:
+    pass
+
+
+async def shutdown(app: web.Application) -> None:
+    pass
+
+
+def create_app(
+    default_cors_options: CORSOptions,
+) -> Tuple[web.Application, Iterable[WebMiddleware]]:
+    app = web.Application()
+    app["prefix"] = "storage/filebrowser"
+    app["api_versions"] = (
+        2,
+        3,
+        4,
+    )
+    app.on_startup.append(init)
+    app.on_shutdown.append(shutdown)
+    cors = aiohttp_cors.setup(app, defaults=default_cors_options)
+    cors.add(app.router.add_route("POST", r"/create", create_or_update_filebrowser))
+    cors.add(app.router.add_route("DELETE", r"/destroy", destroy_filebrowser))
```

Comment:
These regex literals are not used to represent regex literal, so I think it's better to replace with plain string literals.
```suggestion
    cors.add(app.router.add_route("POST", "/create", create_or_update_filebrowser))
    cors.add(app.router.add_route("DELETE", "/destroy", destroy_filebrowser))
```